portfolio/src/main/java/com/google/sps/data/TotalCommentProtos.java

```diff
@@ -0,0 +1,1783 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
This file is auto-generated from the proto file. Please do not add it to the repo. In general, we only want to add the minimal sources from which your portfolio site may be reconstructed.

Reply:
Thank you! Removed the file!